src/style.css

```diff
@@ -1,3 +1,91 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Regular.ttf);
+}
+
+.navigation {
+  display: flex;
+  width: 100vw;
```

Comment:
Don't fix container size (if there is no such a requirement). Here, you've set the width of `.navigation` to `100vw`, which might cause horizontal scrolling if vertical scrollbars are present. It's better to use `width: 100%;` to avoid this issue.